app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -26,7 +26,7 @@ def update(attributes)
 
         # This allows the attributes
         def grants_as_collection(attributes)
-          attributes_collection = attributes[:access_grants_attributes]
+          attributes_collection = Array.wrap(attributes[:access_grants_attributes])
```

Comment:
Yes, that's true, I guess you don't need the method at all, if the attribute isn't found.